worker/create_batches_worker.go

```diff
@@ -393,25 +394,39 @@ func (b *CreateBatchesWorker) Process(message *workers.Msg) {
 	}
 	if len(job.CSVPath) > 0 {
 		err = b.createBatchesUsingCSV(job, isReexecution, dbPageSize)
-		checkErr(l, err)
+		if err != nil {
+			job.TagError(b.MarathonDB, "create_batches_worker", err.Error())
```

Comment:
`create_batches_worker` is repeated several times, maybe create a variable?

Reply:
I create one checkErr method for the CreateBatcheWorker struct to don't repeat the error sentence.